### ISSUE TITLE
Low contrast ratio for disabled elements

### ISSUE THREAD

**Issue by yurii-vasyliev** [canonical/vanilla-framework#4742]
**Describe the bug**

All disabled elements (inputs, buttons, tabs, navigation/pagination items, etc.) have 0.33 opacity as set in `$disabled-element-opacity` variable (which is not overridable). Wish such a low value disabled elements have low contrast ratio.

**To Reproduce**

Steps to reproduce the behaviour:

1. Create any disabled element
2. Check its contrast ratio with browser dev tools or any other tool.

**Expected behaviour**

A minimal required contrast ratio according to AAA standards has to be met.

**Screenshots**

<img width="400" alt="image" src="https://user-images.githubusercontent.com/83575/233589469-2cf56d8d-a2c3-4bd5-847b-35001ab3e8fb.png">

<img width="400" alt="image" src="https://user-images.githubusercontent.com/113896226/233593808-9be203ad-e7f0-4aa2-a41a-fe897801add6.png">

**Comment by lyubomir-popov** [comment]
My understanding of the accessibility aspects of this is that it is more important to clearly indicate the element cannot be interacted with - looking at the guidelines, there's a special [exemption ](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Contrast_(Minimum)#:~:text=Check%20that%20the%20visual%20presentation,immediate%20surrounding%20color(s).
) specifically for disabled elements:
![image](https://user-images.githubusercontent.com/2741678/233606670-d291068a-2045-4937-9d7e-47714562bb40.png)

Making them more contrast can result in people thinking they are active, and being frustrated by the inability to interact with them, so I believe the current styling is appropriate.

Your screenshot raises  another interesting question though - how much disabled is too much? If 80% of the view of this app cannot be interacted with, is disabling it the right UX choice? Or should you explore alternatives like replacing the ocntent with an explanation as to why this part of the UI is not available.
